test/test_tensorboard.py

```diff
@@ -500,6 +505,18 @@ def test_scalar(self):
             res = make_np(np.int64(100000000000))
             self.assertIsInstance(res, np.ndarray) and self.assertEqual(res.shape, (1,))
 
+        def test_numpy_caffe_blob(self):
+            workspace.FeedBlob("testBlob", np.random.randn(1, 3, 64, 64).astype(np.float32))
+            self.assertIsInstance(make_np('testBlob'), np.ndarray)   
+
+        def test_numpy_expect_fail(self):
+            with self.assertRaises(RuntimeError):
+                res = make_np('This_blob_does_not_exist')
+
+        def test_numpy_expect_fail_2(self):
+            with self.assertRaises(NotImplementedError):
+                res = make_np({'pytorch': 1.0})
+
```

Comment:
Looks like the following two tests don't do anything - should they be removed?

Reply:
oh, They should belong to the test of `_prepare_video()`, moving and fixing.

Reply:
skipIfNoCaffe2 is used to prevent mac CI from failing. It needs protobuf for caffe2.workspace to work.
I also added docstring for `_prepare_video`.